frontend/src/components/Map.jsx

```diff
@@ -0,0 +1,91 @@
+import {useMemo, useCallback, useState, useRef, useEffect} from 'react'
+import {GoogleMap} from '@react-google-maps/api'
+import LocationPin from '@/components/LocationPin'
+import SearchBar from '@/components/SearchBar'
+
+
+// post method
+//TODO: check userID, generate tripID
+
+const postTrip = async (trip) => {
+    console.log(JSON.stringify({"trip": trip}))
+    const response = await fetch('http://localhost:8080/trips/1', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({"trip": trip})
+    });
+    const data = await response.json();
+    console.log(data);
+}
+
+
+const Map = (props) => {
+    const [pan, setPan] = useState();
+    const [tripArray, setTripArray] = useState([]);
+    const mapRef = useRef();
+    const center = useMemo( () => ({
+        lat: 33.68,
+        lng: -117.83,
+        // TODO: change to user's location
+    }), []);
+
+    const options = useMemo( () => ({
+        streetViewControl: false,
+        mapTypeControl: false,
+        fullscreenControl: false,
+        clickableIcons: false,
+        mapId: "b6a8170c1c0be23f",
```

Comment:
Why is this Id hardcoded

Reply:
not really a secret thing but ig ill add it to env